### ISSUE TITLE
2.7 to 2.8 upgrade error

### ISSUE THREAD

**Issue by DeadPixhell** [centreon/centreon#5220]
Hi, i have the same problem.

I am trying to update to the last version of centreon, from 2.7.8 to 2.8.8. with yum

I have followed the same documentation as Apsha, and i have the same error.

I have made some investigation on the database, it like the two .sql file of the update 2.7.10 to 2.8.0-beta1 was already executed.

For testing when i have this error i have commented the two SQL file. After doing this i have hit the refresh button.
And now the 2.7.10 to 2.8.0-beta1 was executing in infinite loop, its like the update is not tagged ok and the upgrade loop on this update.

When a have made a test run on a clone couple of weeks ago (2.7.8 to 2.8.6) everythings works fine.

Thanks for Help

a post was made here https://forum.centreon.com/forum/installation-upgrade/144326-update-to-centreon-3-4-error-update-cstg-2-7-10_to_2-8-0-beta1-sql
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 
2.7.8

**OS**: 
centos 6

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Vmware

**Steps to reproduce the issue:**
1. a centreon in 2.7
2. yum update centreon
3. follow the upgrade step
4; it'crash when it's update database

**Comment by niclem** [comment]
Hi, 
The file ../centreon/Update-DB-2.7.10_to_2.8.0-beta1.sql seems to contain a mistake

UPDATE `informations` SET `value` = '2.8.0-beta1' WHERE CONVERT( `informations`.`key` USING utf8 )  = 'version' AND CONVERT ( `informations`.`value` USING utf8 ) = '**2.7.9'** LIMIT 1;

replace 2.7.9 to 2.7.10 resolves the infinite loop for me

**Comment by niclem** [comment]
re Hi,

I find some other constraint error with the script ../centreon/Update-DB-2.8.0_to_2.8.1.sql causing upgrade wizzard to hand

I had to play lines after the 25th on the centreon database.

INSERT INTO `cb_module` (`name`, `libname`, `loading_pos`, `is_bundle`, `is_activated`) VALUES ('Correlation', 'correlation.so', 30, 0, 1);

next click to refresh and the other steps run fine for me

**Comment by kduret** [comment]
Fixed here : f6a97a685967c6a2f81072d5d7e24b34dd1ac465

**Comment by erawor** [comment]
Not works for me, but i found the problem and now works.

`Go to : cd /usr/share/centreon/www/install/sql/centstorage/`

Edit this file : **Update-CSTG-2.7.10_to_2.8.0-beta1.sql**

Comment line 2 and line 5 like that:  add "--" before

```
-- OPTIMIZE Monitoring
-- ALTER TABLE services ADD INDEX last_hard_state_change (last_hard_state_change);

-- Add Timezone in hosts TABLE
-- ALTER TABLE `hosts` ADD COLUMN `timezone` varchar(64) DEFAULT NULL AFTER `statusmap_image`;
```
![migration](https://cloud.githubusercontent.com/assets/28799974/26236196/6fca84e8-3c6f-11e7-8d1f-2994f5c83c0c.PNG)
![sql_modif](https://cloud.githubusercontent.com/assets/28799974/26236198/6fcc31c6-3c6f-11e7-9335-3233336a8216.PNG)
![centreon_ok](https://cloud.githubusercontent.com/assets/28799974/26236197/6fcae9c4-3c6f-11e7-8542-514411773cec.PNG)

**Comment by thiuyendang** [comment]
OK with 2.8.9-4 (migration from 2.7.11 to 2.8.9), CentOS6

**Comment by thiuyendang** [comment]
OK with fix in #5220 isue
